lib/index.js

```diff
@@ -744,6 +744,31 @@ module.exports = class OtpRunner {
     }
   }
 
+  /**
+   * If told to, will trigger the Pelias custom instance webhook to update
+   * the custom GTFS and CSV data
+   */
+  async updatePeliasInstance () {
+    if (this.manifest && this.manifest.updateCustomPelias) {
+      const response = await fetch('http://ec2-3-236-166-78.compute-1.amazonaws.com/webhook', {
+        body: JSON.stringify(this.manifest),
+        headers: {
+          'Accept': 'application/json',
+          // TODO: This absolutely needs to be read from a configuration file!
```

Comment:
That sounds right, so probably from an `env.enc.yml` that we decrypt? Probably the URL webhook should be in that config file too.

Reply:
I was thinking of this, but then wouldn't that file live in the configurations repo? I'm not sure how otp-runner is currently run, but would that be realistic to do?